applications/ShapeOptimizationApplication/python_scripts/analyzer_internal.py

```diff
@@ -80,18 +83,20 @@ def FinalizeAfterOptimizationLoop( self ):
     def __CreateResponseFunctions( specified_responses, model ):
         response_functions = {}
 
-        available_sho_response_functions = ["plane_based_packaging", "mesh_based_packaging"]
-        available_csm_response_functions = ["strain_energy", "mass", "eigenfrequency", "adjoint_local_stress", "adjoint_max_stress"]
+        sho_response_functions = ["plane_based_packaging", "mesh_based_packaging"]

```

Comment:
I actually liked the hint that those are functions that are available or supported and not THE response functions. What about calling them:

```suggestion
        supported_sho_response_functions = ["plane_based_packaging", "mesh_based_packaging"]
```

The other lines would need to be adjusted accordingly. Your call.

Reply:
I think it is obvious that the ones we are using defined are the supported ones.

Reply:
Now i also remember why i removed the "available" or "supported" prefix: In case the StructuralMechanicsApplication is not compiled, those response functions are actually not available. This list contains just the hint in which application the responses can be found.